core/pkg/resultshandling/printer/v2/prettyprinter.go

```diff
@@ -255,8 +255,10 @@ func frameworksScoresToString(frameworks []reportsummary.IFrameworkSummary) stri
 	return ""
 }
 
-func getControlLink(controlID string) string {
-	return fmt.Sprintf("https://hub.armosec.io/docs/%s", strings.ToLower(controlID))
+func GetControlLink(controlID string) string {
```

Comment:
This function should be in the [utils](https://github.com/kubescape/kubescape/tree/dev/core/cautils) package (you can create a new file for this) and not in the printer package.

Reply:
> This function should be in the [utils](https://github.com/kubescape/kubescape/tree/dev/core/cautils) package (you can create a new file for this) and not in the printer package.

Hi @dwertent, 
Made the necessary changes!!